api/routes/activities/index.js

```diff
@@ -1,14 +1,20 @@
+const logger = require('../../logger')('activities route index');
 const defaultActivityModel = require('../../db').models.activity;
 const can = require('../../auth/middleware').can;
 
 module.exports = (app, ActivityModel = defaultActivityModel) => {
+  logger.silly('setting up GET /activities route');
   app.get('/activities', can('view-activities'), async (req, res) => {
+    logger.silly(req, 'handling GET /activities');
     try {
-      const activities = await ActivityModel.fetchAll({
+      const activities = (await ActivityModel.fetchAll({
         columns: ['id', 'name']
-      });
+      })).map(a => ({ id: a.get('id'), name: a.get('name') }));
```

Comment:
Switched to using bookshelf's `get` methods in a few place instead of sending back the raw bookshelf object.  Sending back the bookshelf object is an act of trust that it will serialize the way we want it to.  This way we don't have to trust.  (And also, we can log the pure objects directly.)

Reply:
👍